feat: expose terminal bufnr/winnr in state module (#19)
igorlfs/nvim-dap-view: README.md

<!--toc:start-->
- [nvim-dap-view](#nvim-dap-view)

>   - [Installation](#installation)

Reviewer: Can you keep the indentation with 4 spaces?

Author: Yeah, sorry must have been my markdown formatter

```

If you are using an adapter that does not natively support the `nvim-dap` integrated

> terminal, but you want to use the `nvim-dap-view` terminal to run your debug
> adapter before `nvim-dap` has started a session use the following setup configuration:

Reviewer: ```suggestion
terminal, but you want to use the `nvim-dap-view` terminal anyway, you can set
the following option:
```
